--- a/llava/model/builder.py
+++ b/llava/model/builder.py
@@ -30,7 +30,7 @@
                 tokenizer = AutoTokenizer.from_pretrained(model_path)
                 model = LlavaMistralForCausalLM.from_pretrained(
                     model_path,
-                    low_cpu_mem_usage=False,
+                    low_cpu_mem_usage=True,
                     use_flash_attention_2=False,
                     **kwargs
                 )
